packages/Webkul/Installer/src/Resources/lang/bg/app.php

```diff
@@ -0,0 +1,822 @@
+                'bagisto-info' => 'Създаване на таблиците в базата данни, това може да отнеме няколко минути!',
+                'title'        => 'Инсталация',
+            ],
+
+            'installation-completed' => [
+                'admin-panel'                => 'Администраторски панел',
+                'bagisto-forums'             => 'Bagisto Форум',
+                'customer-panel'             => 'Клиентски панел',
+                'explore-bagisto-extensions' => 'Разгледайте разширенията на Bagisto',
+                'title'                      => 'Инсталацията е завършена',
+                'title-info'                 => 'Bagisto е успешно инсталиран на вашата система.',
+            ],
+
+            'ready-for-installation' => [
+                'create-databsae-table'   => 'създадат нови таблици в базата данни',
+                'install'                 => 'Инсталация',
+                'install-info'            => 'Bagisto за инсталация',
+                'install-info-button'     => 'Кликнете върху бутона по-долу, за да се:',
+                'populate-database-table' => 'запълнят с данни, таблиците в базата данни',
+                'start-installation'      => 'Започнете инсталацията',
+                'title'                   => 'Готов за инсталация',
+            ],
+
+            'start' => [
+                'locale'        => 'Език',
+                'main'          => 'Начало',
+                'select-locale' => 'Избери език',
+                'title'         => 'Вашата инсталация на Bagisto',
+                'welcome-title' => 'Добре дошли в Bagisto!',
+            ],
+
+            'server-requirements' => [
+                'calendar'    => 'Calendar',
+                'ctype'       => 'cType',
+                'curl'        => 'cURL',
+                'dom'         => 'dom',
+                'fileinfo'    => 'fileInfo',
+                'filter'      => 'Filter',
+                'gd'          => 'GD',
+                'hash'        => 'Hash',
+                'intl'        => 'intl',
+                'json'        => 'JSON',
+                'mbstring'    => 'mbstring',
+                'openssl'     => 'openssl',
+                'pcre'        => 'pcre',
+                'pdo'         => 'pdo',
+                'php'         => 'PHP',
+                'php-version' => '8.1 или по-висока',
+                'session'     => 'session',
+                'title'       => 'Системни изисквания',
+                'tokenizer'   => 'tokenizer',
+                'xml'         => 'XML',
+            ],
+
+            'arabic'                   => 'Арабски',
+            'back'                     => 'Назад',
+            'bagisto'                  => 'Bagisto',
+            'bagisto-info'             => 'е Обществен проект от',
+            'bagisto-logo'             => 'Bagisto лого',
+            'bengali'                  => 'Бенгалски',
+            'bulgarian'                => 'Български',
```

Comment:
Here also

The `bulgarian` key should be added in all locales in the installer package.